<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AlertSettingsDialog</class>
 <widget class="QDialog" name="AlertSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>968</width>
    <height>660</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Emergency Information</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>0</height>
         </size>
        </property>
        <property name="title">
         <string>Analog Alarm</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <layout class="QFormLayout" name="formLayout">
           <property name="labelAlignment">
            <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Emergency Alarm</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>ENI Type Select</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Emergency ID</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Alarm Time[s]</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Duration of TX[s]</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Duration of RX[s]</string>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Emergency ENI Send Select</string>
             </property>
            </widget>
           </item>
           <item row="7" column="0">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Emergency Channel</string>
             </property>
            </widget>
           </item>
           <item row="8" column="0">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Emergency Cycle</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="analogEmergencyAlarmCmbx"/>
           </item>
           <item row="1" column="1">
            <widget class="QComboBox" name="analogEniSelectCmbx"/>
           </item>
           <item row="2" column="1">
            <widget class="QComboBox" name="analogEmergencyIdCmbx"/>
           </item>
           <item row="3" column="1">
            <widget class="QComboBox" name="analogAlarmTimeCmbx"/>
           </item>
           <item row="5" column="1">
            <widget class="QComboBox" name="analogRxDurationCmbx"/>
           </item>
           <item row="4" column="1">
            <widget class="QComboBox" name="analogTxDurationCmbx"/>
           </item>
           <item row="6" column="1">
            <widget class="QComboBox" name="analogEniSendSelectCmbx"/>
           </item>
           <item row="7" column="1">
            <widget class="QComboBox" name="analogEmergencyChannelCmbx"/>
           </item>
           <item row="8" column="1">
            <widget class="QComboBox" name="analogEmergencyCycleCmbx"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>0</height>
         </size>
        </property>
        <property name="title">
         <string>Digital Alarm</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QFormLayout" name="formLayout_2">
           <property name="labelAlignment">
            <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>Emergency Alarm</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>Alarm Time[s]</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>Duration of TX[s]</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>Duration of RX[s]</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>Emergency ENI Send Select</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>Emergency Channel</string>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="label_16">
             <property name="text">
              <string>Emergency Cycle</string>
             </property>
            </widget>
           </item>
           <item row="7" column="0">
            <widget class="QLabel" name="label_17">
             <property name="text">
              <string>TG/DMR ID</string>
             </property>
            </widget>
           </item>
           <item row="8" column="0">
            <widget class="QLabel" name="label_18">
             <property name="text">
              <string>Call Type</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="digitalEmergencyAlarmCmbx"/>
           </item>
           <item row="1" column="1">
            <widget class="QComboBox" name="digitalAlarmTimeCmbx"/>
           </item>
           <item row="2" column="1">
            <widget class="QComboBox" name="digitalTxDurationCmbx"/>
           </item>
           <item row="3" column="1">
            <widget class="QComboBox" name="digitalRxDurationCmbx"/>
           </item>
           <item row="4" column="1">
            <widget class="QComboBox" name="digitalEniSendSelectCmbx"/>
           </item>
           <item row="5" column="1">
            <widget class="QComboBox" name="digitalEmergencyChannelCmbx"/>
           </item>
           <item row="6" column="1">
            <widget class="QComboBox" name="digitalEmergencyCycleCmbx"/>
           </item>
           <item row="8" column="1">
            <widget class="QComboBox" name="digitalCallTypeCmbx"/>
           </item>
           <item row="7" column="1">
            <widget class="QLineEdit" name="digitalTgDmrIdTxt"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QCheckBox" name="receiveAlarmChbx">
           <property name="text">
            <string>Receive Alarm</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="qdcSettings">
     <property name="title">
      <string>QDC1200 Settings</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <layout class="QFormLayout" name="formLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="label_26">
          <property name="text">
           <string>Kind</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_27">
          <property name="text">
           <string>Group ID</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_28">
          <property name="text">
           <string>Private ID</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="qdcKindCmbx"/>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="qdcGroupId">
          <property name="maxLength">
           <number>3</number>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="qdcPrivateId">
          <property name="maxLength">
           <number>4</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QGroupBox" name="workAloneD878UVII">
        <property name="minimumSize">
         <size>
          <width>350</width>
          <height>0</height>
         </size>
        </property>
        <property name="title">
         <string>Work Alone</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QFormLayout" name="formLayout_5">
           <property name="labelAlignment">
            <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
           </property>
           <item row="0" column="1">
            <widget class="QComboBox" name="workAloneVoiceSwitchCmbx"/>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_19">
             <property name="text">
              <string>voice switch broadcast</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_20">
             <property name="text">
              <string>area switch broadcast</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_21">
             <property name="text">
              <string>mic switch broadcast</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QComboBox" name="workAloneAreaSwitchCmbx"/>
           </item>
           <item row="2" column="1">
            <widget class="QComboBox" name="workAloneMicSwitchCmbx"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="workAloneD890UV">
        <property name="minimumSize">
         <size>
          <width>350</width>
          <height>0</height>
         </size>
        </property>
        <property name="title">
         <string>Work Alone</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QFormLayout" name="formLayout_6">
           <property name="labelAlignment">
            <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="label_23">
             <property name="text">
              <string>Response Time</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_24">
             <property name="text">
              <string>Warning Time</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_25">
             <property name="text">
              <string>Response</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="workAloneResponseTimeCmbx">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QComboBox" name="workAloneWarningTimeCmbx">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QComboBox" name="workAloneResponseCmbx">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QCheckBox" name="manDownChbx">
           <property name="text">
            <string>Man Down</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QFormLayout" name="formLayout_4">
           <item row="0" column="0">
            <widget class="QLabel" name="label_22">
             <property name="text">
              <string>Man Down Delay[s]</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="manDownDelayCmbx"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QWidget" name="widget_4" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="saveBtn">
              <property name="text">
               <string>OK</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="cancelBtn">
              <property name="text">
               <string>Cancel</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
